third_party/mkl_dnn/mkldnn_v1.BUILD

```diff
@@ -0,0 +1,135 @@
+exports_files(["LICENSE"])
```

Comment:
Could you please explain we why need a seperate file? (I'm not against it. I think it's good for safety. Just curious.)

Reply:
We thought about adding it to same mkldnn.BUILD file, but there were a lot of conditional checks that were complicating dependencies. And we would remove dnnl0.x sometime in the future. So having a separate file would make removal easy.

Reply:
Makes sense. Thank you!